content/articles/coming-soon-landing-page-using-javascript-timers/index.md

```diff
@@ -0,0 +1,403 @@
+- [Cancelling A Timer](#cancelling-a-timer)
+- [Template section](#template-section)
+- [Best practices](#best-practices)
+- [conclusion](#conclusion)
+
+### Javascript Timers Overview
+
+---
+
+A timer is a feature that allows us to run a program at a predetermined time.
+Timers can be used to postpone code execution so that it doesn't finish at the same time as an event or a page loading.
+Timers, for example, can be used to alter your website's advertisement banners at regular intervals or to display a real-time clock, among other things. In JavaScript, there are two timer functions: `setTimeout ()` and `setInterval ()`. Browsers implement timer functions, and their implementations differ from one browser to another. The Node.js runtime also has a native implementation of timers. In browsers, the window interface is the parent of the main timer features. Because the window interface makes its elements available globally in the main javascript scope, setTimeout may be used in the browser console.
+'Timers' is an object in NodeJs that functions like the browser's window interface.
+
+### Code Execution After a Delay
+
+---
+
+The `setTimeout()` function is used to only run a function or a piece of code once after a given amount of time has passed.
+
+**syntanx**
+
+`setTimeout(function, milliseconds)`.
+
+You must give two parameters to utilize this function:A Function parameter that describes the function to be executed, and an optional Delay parameter that determines how long to wait before the function is executed (1 second = 1000 milliseconds).
+
+**Example**
+
+```javascript
+<script>
+function delayGreetings() {
+    alert('Hello World!');
+}
+</script>
+<button type="button" onclick="setTimeout(delayGreetings,2000)">click Me</button>
+```
+
+**when the button is clicked the greetings will be displayed after 2 seconds**
+
+**NOTE :** If the delay parameter is 0 or if it is not specified, the function is executed immediately.
+
+_TASK-1_
+
+Use `setTimeout` function to print the following two messages after their respective delays .
+
+- After 4 seconds, display greetings "Hello FabulousDesigns after 4secs"
+- After 8 seconds, display greetings "Hello FabulousDesigns after 8secs."
+
+**CONSTRAIT :**
+In your solution, you can only define one function, which includes inline functions. As a result, a lot of setTimeout calls will have to use the same code.
+
+**SOLUTION**
+
+- **Task-1** may be solved in a variety of ways, but this is one of them.
+
+```javascript
+  const delayGreetings = (delay) => {
+    console.log("Hello FabulousDesigns after " + delay + " seconds");
+  };
+  setTimeout(delayGreetings, 4 * 1000, 4);
+  setTimeout(delayGreetings, 8 * 1000, 8);
+  ```
+
+#### Explanation
+
+- I gave delayGreetings a delay parameter and utilized the delay argument's value in the displayed message. As a result, depending on the delay value we provide to the function, it might print different messages.

```

Comment:
`delayGreetings` is a function, have some emphasis on it and others where applicable.